enterprise/frontend/src/embedding-sdk/cli/snippets/analytics-dashboard-snippet.ts

```diff
@@ -23,53 +23,55 @@ import { AnalyticsContext } from "./analytics-provider"
 ${imports}
 
 export const AnalyticsDashboard = () => {
-  const {email} = useContext(AnalyticsContext)
+  const {email, themeKey} = useContext(AnalyticsContext)
   const [dashboardId, setDashboardId] = useState(DASHBOARDS[0].id)
 
   const [isCreateQuestion, toggleCreateQuestion] = useReducer((s) => !s, false)
 
   const isDashboard = !isCreateQuestion
 
   return (
-    <div className="analytics-container">
-      <div className="analytics-header">
-        <div>
-          ${userSwitcherEnabled ? "<UserSwitcher />" : ""}
+    <div className={\`analytics-root theme-\${themeKey}\`}>
```

Comment:
Use "hide whitespace changes" to view this diff. It basically just wraps the component with more CSS classes.